### ISSUE TITLE
Could not find 'Skclusive' in 'window' with multiple circuits in server side dashboard demo

### ISSUE THREAD

**Issue by BrunoZell** [skclusive/Skclusive.Material.Component#56]
This problem only occurs in server side Blazor.

This bug is reproducable in the Dashboard demo [Skclusive.Blazor.Dashboard](https://github.com/skclusive/Skclusive.Blazor.Samples/tree/master/Skclusive.Blazor.Dashboard) by running _Dashboard.Server.Host_. The default browser window with the sample page should open without any problems. However, when navigating to the same page in a new tab while leaving the original one open, the second circuit runs into an exception:

```
fail: Microsoft.AspNetCore.Components.Server.Circuits.CircuitHost[111]
      Unhandled exception in circuit 'yIps8o-SOzz4DiBX8NBdOQEaYtW_Pqts_2JpsEtFVr8'.
Microsoft.JSInterop.JSException: Could not find 'Skclusive' in 'window'.
Error: Could not find 'Skclusive' in 'window'.
    at https://localhost:5001/_framework/blazor.server.js:8:30944
    at Array.forEach (<anonymous>)
    at p (https://localhost:5001/_framework/blazor.server.js:8:30904)
    at https://localhost:5001/_framework/blazor.server.js:8:31614
    at new Promise (<anonymous>)
    at e.beginInvokeJSFromDotNet (https://localhost:5001/_framework/blazor.server.js:8:31587)
    at https://localhost:5001/_framework/blazor.server.js:1:20052
    at Array.forEach (<anonymous>)
    at e.invokeClientMethod (https://localhost:5001/_framework/blazor.server.js:1:20022)
    at e.processIncomingData (https://localhost:5001/_framework/blazor.server.js:1:18006)
   at Microsoft.JSInterop.JSRuntime.InvokeWithDefaultCancellation[T](String identifier, Object[] args)
   at Microsoft.JSInterop.JSRuntimeExtensions.InvokeVoidAsync(IJSRuntime jsRuntime, String identifier, Object[] args)
   at Skclusive.Script.DomHelpers.DomHelpers.MoveContentAsync(Nullable`1 source, Nullable`1 target, Nullable`1 targetBody)
   at Skclusive.Material.Core.Portal.OnAfterRenderAsync() in C:\Repos\Skclusive.Material.Component\Core\src\Core\Portal.cs:line 71
   at Skclusive.Core.Component.RenderComponentBase.OnAfterRenderAsync(Boolean firstRender)
   at Skclusive.Core.Component.DisposableComponentBase.OnAfterRenderAsync(Boolean firstRender)
   at Microsoft.AspNetCore.Components.RenderTree.Renderer.GetErrorHandledTask(Task taskToHandle)
```

After reloading the second tab the web app works normally. The problem only occurs when the tab navigating to the web app is newly opened.

This problem is not local to the browser. I have deployed a web app on a server and I get the same behavior accross different devices. Whenever a tab navigates to the web app and another circuit is active, it first runs into an error. Only after refreshing the web app behaves normally.

**Comment by skclusive** [comment]
thanks for reporting this. i have identified the issue and will be fixed.

**Comment by BrunoZell** [comment]
Great, because I couldn't locate it really 😄 What was it?

**Comment by skclusive** [comment]
some stupid stuff i was doing just before the release in Skclusive.Core.Component. static cache which is shared in server rendering. :-( https://github.com/skclusive/Skclusive.Core.Component/blob/master/src/Component/Styled.cs#L10

**Comment by skclusive** [comment]
> await Task.Delay(2000))  haha. need to revisit all those places. 

lately bit occupied with another personal project. and also new interesting component library porting.

**Comment by BrunoZell** [comment]
May I get a hint what the error is so I can fix it myself while there is no official release?

**Comment by skclusive** [comment]
sure. you can clone the Skclusive.Core.Component and edit the reference in Skclusive.Material.Core project.

in Skclusive.Core.Component, update the Styled component as below.

```cs
    public class Styled : StyleComponentBase
    {
        /// <summary>
        /// unique name of the style
        /// </summary>
        [Parameter]
        public string Name { set; get; }

        /// <summary>
        /// ChildContent which contains the style definitions
        /// </summary>
        [Parameter]
        public RenderFragment ChildContent { get; set; }

        protected override void BuildRenderTree(RenderTreeBuilder builder)
        {
            builder.OpenElement(0, "style");
            builder.AddAttribute(1, "data-name", Name);
            builder.AddContent(2, ChildContent);
            builder.CloseElement();
        }
    }
```
i found and alternate approach to avoid style repetition if a project styles are referenced in multiple projects. planning to release part of 3.0.0

**Comment by BrunoZell** [comment]
@skclusive Thank you. I just wanted to let you know that this hasn't fixed my issue. The same error occurs as described. Do you have any other ideas? I'll investigate further on my part.

**Comment by skclusive** [comment]
oh ok. i am going to start working on next release 3.0.0 in a week. would incorporate updates in Skclusive.Core also latest blazor updates coming for .NET 5. this would be the first issue i would be addressing.

**Comment by skclusive** [comment]
@BrunoZell i have released 3.0.2-preview1 release in nuget. Skclusive.Blazor.Samples and Skclusive.Material.Docs develop branches updated with the preview. 

i hope the issue is fixed. could you please verify and update the same? thanks.

**Comment by BrunoZell** [comment]
It works, thank you very much! Best Blazor UI library.
